requirements.txt

```diff
@@ -3,3 +3,5 @@ sphinx>=1.4
 ipykernel
 nbsphinx>=0.4
 nbclean
+requests
+aiohttp
```

Comment:
I don’t see those being used anywhere in the code.

If you’re using some optional feature that depends on those, specify that optional feature instead: `somepackage[web]` or so.

Reply:
:+1:

Reply:
Ok can do.